app/packages/looker/src/overlays/util.ts

```
@@ -21,40 +21,39 @@ export const t = (state: BaseState, x: number, y: number): Coordinates => {
+    switch (typeof obj) {
+      case "number":
+        bytes += 8;
+        break;
+      case "string":
+        bytes += obj.length * 2;
+        break;
+      case "boolean":
+        bytes += 4;
+        break;
+      case "object":
+        var objClass = Object.prototype.toString.call(obj).slice(8, -1);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)